README.md

```diff
@@ -8,12 +8,15 @@ This repo provides basic tuning scripts with support for specific models. The re
 ## Installation
 
 ```
-pip install -r requirements.txt
-pip install -U datasets
 pip install -e .
 ```
 
-> Note: If you wish to use [FlashAttention](https://github.com/Dao-AILab/flash-attention), then you need to install these requirements: `pip install -r flashattn_requirements`. [FlashAttention](https://github.com/Dao-AILab/flash-attention) requires the [CUDA Toolit](https://developer.nvidia.com/cuda-toolkit) to be pre-installed.
+> Note: After installing, if you wish to use [FlashAttention](https://github.com/Dao-AILab/flash-attention), then you need to install these requirements:
+'''
+pip install -e ."[dev]"
+pip install -e ."[flash-attn]
```

Comment:
Perhaps drop double quotes to be consistent with the Dockerfile
```suggestion
pip install -e .[dev]
pip install -e .[flash-attn]
```

Reply:
Sure, thanks Ted!  I'm going to do some additional testing this morning, make sure it all looks good...

Reply:
OK - I think I've got it - so on Red Hat linux, it installs fine with
```
pip install -e .[dev]
```
but on Mac I have to put single or double quotes around it.  The quotes also work in Linux, so for simplicity I've changed all the optional install occurrences (README.md, to this format:
```
pip install -e ".[dev]"
```
I think it's good now, and the toml file is now able to build a wheel file that installs on both Mac and Red Hat.